OsmAnd-java/src/main/java/net/osmand/IndexConstants.java

```diff
@@ -63,6 +63,7 @@
 	public static final String MAP_MARKERS_INDEX_DIR = "/map markers";
 	public static final String GPX_RECORDED_INDEX_DIR = GPX_INDEX_DIR + "rec/";
 	public static final String GPX_IMPORT_DIR = GPX_INDEX_DIR + "import/";
+	public static final String GPX_FROM_OBF_DIR = GPX_INDEX_DIR + "obf/";
```

Comment:
bad name, doesn't mean anything for user. it's better to use "travel" cause they are part of it or "maps" if travel doesn't work